lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
+  attr_accessor :cells
+
+  def initialize(board = nil)
+    @cells = board || Array.new(9, " ") 
+
+  end
+
+  def reset!
+    self.cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{self.cells[0]} | #{self.cells[1]} | #{self.cells[2]} "
+    puts "-----------"
+    puts " #{self.cells[3]} | #{self.cells[4]} | #{self.cells[5]} "
+    puts "-----------"
+    puts " #{self.cells[6]} | #{self.cells[7]} | #{self.cells[8]} "
+  end
+
+  def position(input)
+    self.cells[input.to_i - 1]
+  end
+
+  def full?
+    self.cells.include?(" ") ? false : true
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

`cells.none? {|cell| cell == " " || cell == nil}`